docs/tutorial/tasks.md

```diff
@@ -210,12 +200,9 @@ Average open issues per user 💌 : 2.27
 
 </div>
 
-## Subflows
-
-Not only can you call tasks within a flow, but you can also call other flows!
-Child flows are called [subflows](/concepts/flows/#composing-flows) and allow you to efficiently manage, track, and version common multi-task logic.
+## Organize multiple tasks
 
-Subflows are a great way to organize your workflows and offer more visibility within the UI.
+Subflows are a great way to organize multiple tasks and offer more visibility within the UI.
```

Comment:
```suggestion
Subflows are a great way to organize multiple workflows and offer more visibility within the UI.
```

A subflow doesn't necessarily have multiple tasks. Probably makes more sense to use a generic term such as "workflows" instead of the Prefect-specific tasks here.